src/backend/web/validation.js

```diff
@@ -0,0 +1,52 @@
+const Ajv = require('ajv');
+
+const ajv = new Ajv({ allErrors: true });
+
+const validateHttpSchemes = function (schema, uri) {
```

Comment:
Add a comment above this indicating we expect only URI strings starting with `http://` or `https://